vth_scale: 0.975
vx_scale: 1.0
vy_scale: 1.0
sendTransform: 0
